getting-started/what-you-need.md

```diff
@@ -1,6 +1,6 @@
 # What You Need
 
-To get started, you'll need a text editor and the [ponyc](https://github.com/ponylang/ponyc) compiler. That's it.
+To get started, you'll need a text editor and the [ponyc](https://github.com/ponylang/ponyc) compiler. Or if you are on a not supported platform or don't want to install the compiler you can use the [Pony's Playground](https://playground.ponylang.org/)
```

Comment:
The way you motivate mentioning the playground, one could also mention the `ponylang/ponyc` docker container. What do you think?

Reply:
Why not. The advantage of the playground is that it can be used by anyone. How would advertise the container? Adding a link to the pony repository?

Reply:
The first link points to how to install, which includes how to use docker. So I don't think it is necessary here